### DISCUSSION TITLE
大佬为啥不推荐使用npm，我现在项目用的全是npm

### DISCUSSION THREAD

**Discussion by GaoJianXin520** [shengxinjing/it-roadmap#27] [提问]
![image](https://user-images.githubusercontent.com/37009488/145056454-9877543b-3a1d-487b-acc9-149ab0aa9a75.png)

**Comment by shengxinjing** [comment]
倒不是完全不推荐，npm现在有很多问题，yarn会稍微好用一些，pnpm通过软连的方式 ，把node_modules的体积打下来了，所以我比较推荐这俩

**Reply by GaoJianXin520** [reply]
好的，谢谢解惑

**Comment by WalkAlone0325** [comment]
建议尝试切换pnpm吧，vue生态和很多的github相关都切换到pnpm了，早学不吃亏

**Reply by GaoJianXin520** [reply]
收到
